Description: fixes compilation with GCC-6
 return type must be Translator* and can't be boolean
Author: Damyan Ivanov <dmn@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=811619

--- a/src/translator_manager.cpp
+++ b/src/translator_manager.cpp
@@ -176,11 +176,11 @@ Translator *TranslatorManager::getTestDi
 		isDataOk = isDataOk && tr->init(tr->BG_EN, string(string(prop->getDirectory()) + prop->getString("data")).c_str());
 	} else {
 		cerr << "TranslatorManager::getTestDictionaryObject - Wrong description file: " << testDictionaries[index].fileName << "\n";
-		return false;
+		return NULL;
 	}
 	if (!isDataOk) {
 		cerr << "TranslatorManager::getTestDictionaryObject - Problem initialazing dictionary: " << testDictionaries[index].fileName << "\n";
-		return false;
+		return NULL;
 	}
 	char c[16];
 	sprintf(c, "%d", level);
